runtime/compiler/control/MethodToBeCompiled.cpp

```diff
@@ -139,19 +147,21 @@ TR_MethodToBeCompiled::setAotCodeToBeRelocated(const void *m)
    _optimizationPlan->setIsAotLoad(m!=0);
    }
 
+#if defined(JITSERVER_SUPPORT)
 uint64_t 
 TR_MethodToBeCompiled::getClientUID() const
    {
    return _stream->getClientId();
    }
 
 void
-TR_MethodToBeCompiled::cleanupJITaaS()
+TR_MethodToBeCompiled::cleanupJITServer()
```

Comment:
I would like a more descriptive name for this function, but cannot decide on something specific.
Maybe: `cleanupJITServerAllocations` or `cleanupJITServerData`

Reply:
Maybe `freeJITServerAllocations` which shows what this functions does.